src/models/Control.php

```diff
@@ -1,6 +1,12 @@
 <?hh // strict
 
 class Control extends Model {
+
+  protected static string $MC_KEY = 'control:';
+
+  protected static Map<string, string>
+    $MC_KEYS = Map {"ALL_ACTIVITY" => "activity"};
```

Comment:
This is a nitpick, but here and wherever you use double quotes for strings, convert them to single quotes. We only use double quotes when we explicitly want to have variables evaluated in strings.